internal/retry/config.go

```diff
@@ -130,11 +130,18 @@ var (
 	BoTxnLockFast = NewConfig(txnLockFastName, &metrics.BackoffHistogramLockFast, NewBackoffFnCfg(2, 3000, EqualJitter), tikverr.ErrResolveLockTimeout)
 )
 
-var isSleepExcluded = map[string]struct{}{
-	BoTiKVServerBusy.name: {},
+var isSleepExcluded = map[string]int{
+	BoTiKVServerBusy.name: 600000, // The max excluded limit is 10min.
```

Comment:
It's a bit too far to add a new limit of it, I'm afraid that someday it will be required to be configurable.

Reply:
This could only be used in some extreme cases like the service is unavailable but the leader is still valid.